spec/index.html

```diff
@@ -365,6 +380,19 @@ <h3>Terms defined by this specification</h3>
       <dd>A tuple composed of <a>subject</a>, <a>predicate</a>, <a>object</a>, and <a>graph name</a>.
         This is a generalization of an <a>RDF triple</a> along with a <a>graph name</a>.
       </dd>
+      <dt><dfn class="export">canonicalization function</dfn></dt>
+      <dd>A <a>canonicalization function</a> maps <a>RDF datasets</a>
+        into <a data-cite="RDF11-CONCEPTS#dfn-dataset-isomorphism">isomorphic datasets</a>.
```

Comment:
```suggestion
        into <a data-cite="RDF11-CONCEPTS#dfn-dataset-isomorphism">isomorphic datasets</a> [[!RDF11-CONCEPTS]].
```

I think it is good practice to put an explicit reference; the reader sees at once that this is defined in an RDF specification, and it is on our own.